compatibility/bazel_tools/daml_script/daml_script.bzl

```diff
@@ -66,43 +70,80 @@ def daml_script_test(compiler_version, runner_version):
         server = "@daml-sdk-{version}//:sandbox-on-x".format(version = runner_version)
         server_args = ["--participant", "participant-id=sandbox,port=6865"]
         server_files = []
-        runner_files = ["$(rootpath {})".format(compiled_dar)]
+        server_files_prefix = ""
     else:
         server = daml_runner
         server_args = ["sandbox"]
         server_files = ["$(rootpath {})".format(compiled_dar)]
-        runner_files = []
+        server_files_prefix = "--dar=" if use_canton else ""
 
-    client_server_test(
-        name = "daml-script-test-compiler-{compiler_version}-runner-{runner_version}".format(
-            compiler_version = version_to_name(compiler_version),
-            runner_version = version_to_name(runner_version),
+    native.genrule(
```

Comment:
I don’t quite understand why this needs to be a genrule instead of making the sh_binary accept an argument to figure out what it should do so it can be a non-generate bash file.

Reply:
I think the advantage of generating it is we don't have to rely on positional arguments. Not a very strong argument tho.

Reply:
happy either way your choice